Week1/index.js

```diff
@@ -0,0 +1,245 @@
+    'Dependent Territory of the UK',
+    'Elisabeth II',
+    'FK',
+    504,
+  ],
+  [
+    9830,
+    'Hungary',
+    'Europe',
+    093745,
+    93030.0,
+    19189088,
+    10043200,
+    71.4,
+    48267.0,
+    45914.0,
+    'Magyarország',
+    'Republic',
+    'Ferenc Mádl',
+    'HU',
+    09,
+  ],
+  [
+    7324,
+    'Indonesia',
+    'Asia',
+    8406386,
+    1904569.0,
+    1945,
+    212107000,
+    68.0,
+    84982.0,
+    215002.0,
+    'Indonesia',
+    'Republic',
+    'Abdurrahman Wahid',
+    'ID',
+    9583,
+  ],
+];
+
+db.query(insertIntoCountriesTable, [countries], function(error) {
+  if (error) throw error;
+  console.log('data inserted into country table');
+});
+
+// insert data into cities table.
+const insertIntoCities = 'INSERT INTO cities (id,name,Country_Code,District,Population) VALUES?';
+let values = [
+  [1, 'Kabul', 134, 'yemen', 9780000],
+  [2, 'Kandahar', 7890708, 'Afghanistan', 76896],
+  [3, 'Herat', 79987, 'yemen', 886800],
+  [4, 'Clearwater', 797, 'Florida', 99936],
+  [5, 'Midland', 9879, 'Texas', 98293],
+  [6, 'Davenport', 87970, 'Iowa', 98256],
+  [7, 'Amsterdam', 820000, 'Netherlands', 64646],
+  [8, 'The Hague', 24234, 'Netherlands', 353525],
+  [9, 'Rotterdam', 325325, 'Netherlands', 45335],
+  [10, 'Utrecht', 35235, 'Netherlands', 435352],
+  [90, 'Amsterdam', 80000, 'Netherlands', 820000],
+  [69, 'The Hague', 00, 'Netherlands', 510000],
+];
+
+db.query(insertIntoCities, [values], function(error) {
+  if (error) throw error;
+  console.log('data inserted into city table');
+});
+
+// countries with population greater than 8000000.
+db.query('SELECT Name, Population FROM countries WHERE Population > 8000000 ', function(
+  error,
+  resultQuery1,
+) {
+  if (error) throw error;
+  console.log(JSON.parse(JSON.stringify(resultQuery1)));
+});
+
+// countries that has Land in their names.
+db.query('SELECT Name FROM countries WHERE Name like "%land%" ', function(error, resultQuery2) {
+  if (error) throw error;
+
+  console.log(JSON.parse(JSON.stringify(resultQuery2)));
+});
+//cities where population between 500000 and 1000000.
+db.query(
+  'SELECT name, Population FROM cities WHERE Population BETWEEN 500000 AND 1000000',
+  function(error, resultQuery3) {
+    if (error) throw error;
+    console.log(JSON.parse(JSON.stringify(resultQuery3)));
+  },
+);
+
+//countries in continent Europe.
+db.query('SELECT Name FROM countries WHERE Continent="Europe"', function(error, resultQuery4) {
+  if (error) throw error;
+  console.log(JSON.parse(JSON.stringify(resultQuery4)));
+});
+
+//List all the countries in the descending order based on their surface areas.
+db.query('SELECT Name,   Surface_Area FROM countries ORDER BY  Surface_Area DESC ', function(
+  error,
+  resultQuery5,
+) {
+  if (error) throw error;
+  console.log(JSON.parse(JSON.stringify(resultQuery5)));
+});
+
+//names of all the cities in the Netherlands
+db.query('SELECT name  FROM cities WHERE Country_Code="Netherlands"'),
```

Comment:
This is the correct query:
`SELECT name  FROM cities WHERE District="Netherlands"`